### PR TITLE
Adding versioning via changesets

### PR DESCRIPTION
**Pull request by scme0**
Like always, it's very difficult to test this without going for it and fixing the bugs. 😅

### CODE REVIEW

File: .github/workflows/build-and-push-container-image.yml [hunk 1]
```diff
@@ -2,41 +2,62 @@ name: Build and Push Container Image
...
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
+        
 
       - name: Build and push
         uses: docker/build-push-action@v5
         with:
           push: true
-          tags: octopusdeploy/nfs-server:latest
```
Comment (scme0) [inline .github/workflows/build-and-push-container-image.yml:39]: We can publish all tags in a single action including to the two different registries 👍

File: .github/workflows/build-and-push-container-image.yml [hunk 2]
```diff
@@ -2,41 +2,62 @@ name: Build and Push Container Image
...
+    - master
 
 jobs:
   build:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Is Versioning Commit
+        run: |
+            echo "IS_VERSIONING_COMMIT=$(startsWith(github.event.commits[0].message, 'Version Kubernetes Agent Chart'))""
```
Comment (flin-8) [inline .github/workflows/build-and-push-container-image.yml]: name -> `Version NFS Server Chart`

File: .github/workflows/build-and-push-container-image.yml [hunk 3]
```diff
@@ -2,41 +2,62 @@ name: Build and Push Container Image
...
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
+        
 
       - name: Build and push
         uses: docker/build-push-action@v5
         with:
           push: true
-          tags: octopusdeploy/nfs-server:latest
+          tags: ${{ steps.tags.outputs.TAGS }}
```
Comment (flin-8) [inline .github/workflows/build-and-push-container-image.yml]: Let's comment out this step and just print the computed values/tags first

**Review by flin-8 — APPROVED** [review]
LGTM + comments

**Pull request merged**